src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.ts

```diff
@@ -10,14 +14,49 @@ import { StudentExamService } from 'app/exam/manage/student-exams/student-exam.s
 export class StudentExamDetailComponent implements OnInit {
     courseId: number;
     studentExam: StudentExam;
+    course: Course;
+    student: User;
 
-    constructor(private route: ActivatedRoute, private studentExamService: StudentExamService) {}
+    constructor(private route: ActivatedRoute, private studentExamService: StudentExamService, private courseService: CourseManagementService) {}
 
     /**
      * Initialize the courseId and studentExam
      */
     ngOnInit(): void {
+        this.loadAll();
+    }
+
+    loadAll() {
```

Comment:
Please add documentation to all public methods or make them private (I think this one can be private)

Reply:
ist now private